blog_system/settings.py

```diff
@@ -0,0 +1,133 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+from django.conf.global_settings import AUTH_USER_MODEL
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-9t!x9-5m1duf6bzko@3p4mag4#exu&m(t67$v5m=i4&uepn*h4"

```

Comment:
The `SECRET_KEY` is hardcoded in the settings file, which is a security risk. It's recommended to load the secret key from an environment variable or a separate configuration file that is not included in version control.